hack/python-sdk/main.go

```diff
@@ -23,26 +23,48 @@ import (
 	"strings"
 
 	"github.com/go-openapi/spec"
+	mxJob "github.com/kubeflow/tf-operator/pkg/apis/mxnet/v1"
+	pytorchJob "github.com/kubeflow/tf-operator/pkg/apis/pytorch/v1"
 	tfjob "github.com/kubeflow/tf-operator/pkg/apis/tensorflow/v1"
+	xgboostJob "github.com/kubeflow/tf-operator/pkg/apis/xgboost/v1"
 	"k8s.io/klog"
 	"k8s.io/kube-openapi/pkg/common"
 )
 
 // Generate OpenAPI spec definitions for TFJob Resource
 func main() {
-	if len(os.Args) <= 1 {
-		klog.Fatal("Supply a version")
+	if len(os.Args) <= 2 {
+		klog.Fatal("Supply a framework and version")
 	}
-	version := os.Args[1]
+	framework := os.Args[1]
+	version := os.Args[2]
 	if !strings.HasPrefix(version, "v") {
 		version = "v" + version
 	}
-	oAPIDefs := tfjob.GetOpenAPIDefinitions(func(name string) spec.Ref {
-		return spec.MustCreateRef("#/definitions/" + common.EscapeJsonPointer(swaggify(name)))
-	})
```

Comment:
@Jeffwan We might need to generate OpenAPI Definition for our APIs to avoid corev1 definition as you mentioned here: https://github.com/kubeflow/tf-operator/pull/1389#issue-722027433.
Then, our Swagger will have only Training related APIs.
What do you think ?

Reply:
@andreyvelich 

Yeah. I think this is the problem. Not sure what corev1 are populated. Do you have any ideas? Seems it's the something is misconfigured in this `hack/python-sdk/main.go`?

Reply:
@andreyvelich

Yeah. I think this is the problem. Not sure what corev1 are populated. Do you have any ideas? Seems something is misconfigured in this hack/python-sdk/main.go?

Reply:
Do we need to use `code-generator` to generate it ? Similar to this: https://github.com/kubeflow/katib/blob/master/hack/update-openapigen.sh#L49-L53.

Reply:
Yeah. Same tool. I update the `--input-dirs` to exclude those unrelated files. Thanks @andreyvelich